### ISSUE TITLE
global.URL.createObjectURL is not a function

### ISSUE THREAD

**Issue by bryanSME** [openlayers/openlayers#11018]
**Describe the bug**
Jest fails to start when importing Heatmap `import HeatmapLayer from 'ol/layer/Heatmap`. **Screenshot of the error below**  I'm not entirely sure this is an OpenLayers bug but it seemed odd since it started in v6.0.0 and only occurs with heatmap.  Jest works fine with everything else.

**To Reproduce**
run any jest test that imports the file 

#### Instantiating a heatmap. For simplicity sake, I'm not using the heatmap in this screenshot, just Instantiating it which causes the error.
![image](https://user-images.githubusercontent.com/16226280/81343802-a87b7480-9083-11ea-86bb-d84ac11249d7.png)

### The Jest Error
![image](https://user-images.githubusercontent.com/16226280/81343898-ccd75100-9083-11ea-9729-339f8da2e048.png)

**Expected behavior**
Jest should run without choking

**Comment by mike-000** [comment]
See https://stackoverflow.com/questions/52968969/jest-url-createobjecturl-is-not-a-function
You might have a similar issue with KML where use of URL is documented https://openlayers.org/en/latest/apidoc/module-ol_format_KML-KML.html

**Comment by bryanSME** [comment]
@mike-000 thanks for the find.  I'm looking into this now.

**Comment by bryanSME** [comment]
@mike-000 

thanks for the help.  I got it working with HeatMap.  Most of the solutions didn't work for me but this one did.

In my `jest.config.js` file, I added the following to the global. The fat arrow isn't needed, `createObjectURL(): {}` should also be ok. No additional code was needed in the test itself.

![image](https://user-images.githubusercontent.com/16226280/81561276-f2ed4180-9360-11ea-871b-a430b5fadc7d.png)

Here are the [Jest global docs](https://jestjs.io/docs/en/configuration#globals-object)
